libs/features/georoutingv2/georoutingv2.js

```diff
@@ -228,13 +228,7 @@ function buildContent(currentPage, locale, geoData, locales) {
 async function getDetails(currentPage, localeMatches, geoData) {
   const availableLocales = await getAvailableLocales(localeMatches);
   if (!availableLocales.length) return null;
-  const { innerWidth } = window;
-  let svgDiv = null;
-  if (innerWidth < 480) {
-    const { default: getMobileBg } = await import('./getMobileBg.js');
-    svgDiv = createTag('div', { class: 'georouting-bg' }, getMobileBg());
-  }
-  const georoutingWrapper = createTag('div', { class: 'georouting-wrapper fragment', style: 'display:none;' }, svgDiv);
+  const georoutingWrapper = createTag('div', { class: 'georouting-wrapper fragment', style: 'display:none;' });
```

Comment:
Unrelated to this PR, but I see the display declaration is being kept and overridden by CSS with `!important`. Does this need to stay like this? Could we use a modifier class instead of adding the style via JS? Could we remove the style attribute at a later point? It's just a bit confusing to see
<img width="1728" alt="Screenshot 2024-09-17 at 16 36 05" src="https://github.com/user-attachments/assets/2b50d0eb-2cd6-443b-ad53-c42a5d2d6388">

Reply:
@overmyheadandbody This  would currently be required as the getDetails function that fetches and adds the modal to the DOM is called before showModal that fetches the georoutingv2.css and till that time the modal needs to be hidden. We would either need to add this as class to a different css file that gets loaded before and then apply this class here.
We can probably explore this as a part of a different PR? Could you share your thoughts on the same?

@narcis-radu @mokimo @robert-bogos Could you share your inputs on the same?